templates/flavors/system-assigned-identity/system-assigned-identity-md.yaml

```diff
@@ -1,76 +0,0 @@
----
-apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
nice, I noticed in #833 that there was a duplication of the md kubeadmConfigTemplate and was going to open a PR for it later 👍